internal/clients/nightfall/nightfall.go

```diff
@@ -195,11 +194,16 @@ func createCommentsFromScanResp(
 	return comments
 }
 
-func isFindingInTokenExclusionSet(fragment string, tokenExclusionSet *strset.Set) bool {
-	if tokenExclusionSet == nil {
+func isFindingInTokenExclusionList(fragment string, tokenExclusionList []string) bool {
+	if tokenExclusionList == nil {
 		return false
 	}
-	return tokenExclusionSet.Has(fragment)
+	for _, pattern := range tokenExclusionList {
+		if matched, _ := regexp.MatchString(pattern, fragment); matched {
```

Comment:
Seems like we should be able to use the [matchRegex](https://github.com/nightfallai/jenkins_test/pull/11/files#diff-d8a0a87ad0f085289ed44c2ba1206b24R361) func from your other diff

Reply:
yup i'll make sure to rebase